### DISCUSSION TITLE
Let's say I have a .tiff images with 8 total bands each. When I feed these images to train my yolo model, what bands are used?

### DISCUSSION THREAD

**Discussion by 7f0c** [ultralytics/ultralytics#6651] [Q&A]
Are all the bands used? or only a portion of them?

**Comment by glenn-jocher** [comment]
@7f0c hello! YOLOv8, like other YOLO models, is designed to work with standard 3-channel RGB images. When you provide multi-band images such as .tiff with 8 bands, the model will not automatically use all the bands. Typically, only the first three bands are interpreted as the RGB channels, and the additional bands are not used in the training process.

If you want to utilize all the bands of your multi-spectral images, you would need to modify the model's input layer to accept the additional channels and adjust the data preprocessing accordingly. This would require a custom implementation beyond the standard usage of YOLOv8. Keep in mind that such modifications can be complex and may require a deep understanding of the model architecture and data handling. 🛠️
